WooCommerce/Classes/ViewRelated/Orders/Order Details/Order Summary Section/SummaryTableViewCell.swift

```diff
@@ -80,6 +31,7 @@ final class SummaryTableViewCell: UITableViewCell {
     func configure(_ viewModel: SummaryTableViewCellViewModel) {
         titleLabel.text = viewModel.billedPersonName
         subtitleLabel.text = viewModel.subtitle
+        salesChannelLabel.text = viewModel.formattedSalesChannel
```

Comment:
nit: might we want to hide the label if the text is nil? since this is in a stack view, if the stack view has a non-zero spacing (from the xib it looks like it has 4px spacing), the spacing still takes up space even if the label has zero width from nil text when it's not hidden.

Reply:
```suggestion
        salesChannelLabel.text = viewModel.formattedSalesChannel
        salesChannelLabel.isHidden = salesChannelLabel.text == nil
```

This fixes the order date not taking up the horizontal space when the sales channel label isn't shown, which is more obvious in large font sizes:

![Simulator Screenshot - iPad (A16) - 2025-07-02 at 11 12 59](https://github.com/user-attachments/assets/ec03ec71-f74b-4f57-a00e-0d28fea64fc1)

Reply:
> if the stack view has a non-zero spacing (from the xib it looks like it has 4px spacing), the spacing still takes up space even if the label has zero width from nil text when it's not hidden.

TIL. Thanks for sharing the solution. Updated on https://github.com/woocommerce/woocommerce-ios/pull/15857/commits/70df74e9a5534aa8b9756d3b23ea781519b6c038